source/plugin/commands/arguments.rst

```diff
@@ -25,13 +25,15 @@ it with :javadoc:`CommandContext#getOne(String)`. Optional and weak arguments ma
 
 Many of the parsers may return more than one object (e.g. multiple players with a matching username). In that case, you
 must use the :javadoc:`CommandContext#getAll(String)` method to get the ``Collection`` of possible matches.
-**Otherwise, the context object will throw an exception!**
+**Otherwise, the context object will throw an exception!** Please make sure that you make a conscious decision whether
```

Comment:
Make this point a separate paragraph. Something like:

> When creating a command, consider whether the argument could return more than multiple values, for example, whether a player argument could support multiple players when using a selector...

Reply:
> return more than multiple values

Thats probably overkill... 
I decided to use `return multiple values` if I should use `return more than one value`  leave a comment below.

Reply:
Ha, yeah, oops. Multiple is fine, IMO